.reek

```diff
@@ -63,6 +63,7 @@ TooManyMethods:
     - OpenidConnectAuthorizeForm
     - Idv::Session
     - User
+    - Idv::ProfileStep
```

Comment:
I've opened an issue to refactor this later because I wanted to keep this PR as small as possible.